Modules/SubModule_ASL/xASL_wrp_VisualQC_ASL.m

```diff
@@ -286,6 +334,14 @@ function xASL_wrp_VisualQC_ASL(x)
     end
 end
 
+% Delete the temporary contours if they exist
+if exist(fullfile(x.SESSIONDIR,'pGMC.nii'),'file')
```

Comment:
`xASL_delete` already checks for existance, so you can remove the `if exist` lines